app/routes/questionnaire.py

```diff
@@ -394,6 +395,32 @@ def get_view_submitted_response(schema, questionnaire_store):
     )
 
 
+@post_submission_blueprint.route("download-pdf", methods=["GET"])
+@with_questionnaire_store
+@with_schema
+def download_pdf(schema, questionnaire_store):
+
+    metadata = get_metadata(user=current_user)
+
+    filename = "{}_{}.pdf".format(metadata.get("eq_id"), metadata.get("form_type"))
```

Comment:
This would have to be based on something that always exists, i.e schema_name or the attrs that will exist for business.

Reply:
> This would have to be based on something that always exists, i.e schema_name or the attrs that will exist for business.

Fine for the spike. In addition this is the requirements and v2 implementation. However, I am surprised at this as both values are internal and probably shouldn't be exposed to the user. Will require some business analysis when we take it forward.